copilot_helper.py

```diff
@@ -46,6 +46,4 @@ def copilot_helper():
 copilot_helper.add_command(waf_commands)
 
 if __name__ == "__main__":
-    # TODO: We should probably wrap this to catch a botocore.exceptions.UnauthorizedSSOTokenError to
```

Comment:
Does this not need to be done anymore?

Reply:
There's a ticket for doing this in a slightly different way, leveraging Lawrence's code.